packages/block-library/src/post-content/block.json

```diff
@@ -8,7 +8,8 @@
 	],
 	"supports": {
 		"align": [ "wide", "full" ],
-		"html": false
+		"html": false,
+		"__experimentalLayout": true
```

Comment:
I think there's also a change that is needed in "edit" function to pass the right layout to `InnerBlocks` ideally this is absorbed by the hook as I comment in the initial PR but we lack the filters and hooks to do so for the moment.